deploy/assisted-service.yaml

```diff
@@ -99,6 +99,31 @@ spec:
                 secretKeyRef:
                   key: endpoint
                   name: assisted-installer-s3
+            - name: AWS_SECRET_ACCESS_KEY_PUBLIC
```

Comment:
Do you need to update https://github.com/openshift/assisted-service/blob/master/openshift/template.yaml as well?